tfexec/version_test.go

```diff
@@ -300,16 +300,16 @@ func TestExperimentsEnabled(t *testing.T) {
 		tfVersion     *version.Version
 		expectedError error
 	}{
-		"experiments-enabled-in-1.9.0-alpha20240404": {
-			tfVersion: version.Must(version.NewVersion(testutil.Alpha_v1_9)),
+		fmt.Sprintf("experiments-enabled-in-%s", testutil.Latest_Alpha_v1_9): {
```

Comment:
```suggestion
		"experiments-enabled-in-alphas": {
```

I think removing the exact version from the map key will make it more readable